### PR TITLE
preparing migration to thoth-zuul-psi

### PR DESCRIPTION
**Pull request by goern**
Depends-On: https://github.com/thoth-station/zuul-infra/pull/11

and preparing the migration to PSI cluster

### CODE REVIEW

**Comment by goern** [comment]
recheck

**Review by leonjia0112 — COMMENTED** [review]
Is this config file current supporting both upshift and psi cluster?
If it does, should we open an issue to update this file once we finish the migration.

**Comment by goern** [comment]
Ja it's supporting both

**Comment by leonjia0112** [comment]
Cool, I think it looks good for now. Just need another update later.

**Review by leonjia0112 — APPROVED** [review]

**Comment by goern** [comment]
@harshad16 need your approving review to have zuul merge it ;)

**Review by harshad16 — APPROVED** [review]
LGTM

File: nodepool/thoth.yaml [hunk 1]
```diff
@@ -16,19 +16,38 @@
...
-            - name: "thoth-ops"
-              image: "thoth-ops:latest"
+
+    - name: "thoth-zuul-psi"
+      driver: openshiftpods
+      context: "thoth-zuul/paas-psi-redhat-com:443/system:serviceaccount:thoth-zuul:nodepool"
+      pools:
+        - name: "thoth-zuul"
+          labels:
+            - name: "thoth-coala"
```
Comment (leonjia0112) [inline nodepool/thoth.yaml:38]: Will it be better in naming if change this name to `thoth-zuul-psi` like above?

**Pull request merged**